docs/source/examples/spot-jobs.rst

```diff
@@ -141,21 +165,23 @@ As HuggingFace has built-in support for periodically checkpointing, we only need
 .. note::
   You may also refer to another example `here <https://github.com/skypilot-org/skypilot/tree/master/examples/spot/resnet_ddp>`_ for periodically checkpointing with PyTorch.
 
-With the above changes, you are ready to launch a spot job with ``sky spot launch``!
+With the above changes, the managed spot job can now resume training after preemption with ``sky spot launch``! We can enjoy the benefits of
+cost savings from spot instances without worrying about preemption or losing progress.
 
 .. code-block:: console
 
     $ sky spot launch -n bert-qa bert_qa.yaml
 
-SkyPilot will launch and start monitoring the spot job. When a preemption happens, SkyPilot will automatically
-search for resources across regions and clouds to re-launch the job. 
 
 .. note::
-  To identify the same job across multiple recoveries, use the environment variable :code:`$SKYPILOT_RUN_ID` in the task's
-  :code:`run` commands or directly in the program itself (e.g., access via :code:`os.environ` and pass to Weights & Biases
-  for tracking purposes in your training script). It is made available to the task whenever it is invoked. The run ID
-  is kept identical across all recoveries of the same spot job.
+  The environment variable :code:`$SKYPILOT_RUN_ID` can be used to identify the same job, i.e., it is kept identical across all
```

Comment:
We might want to move this note to right below the YAML code block for better reference?
otherwise the below note will come first
```
You may also refer to another example [here](https://github.com/skypilot-org/skypilot/tree/master/examples/spot/resnet_ddp) for periodically checkpointing with PyTorch.
```

Reply:
Thanks! I reordered/rephrased the text and remove the note box for the following line to avoid too many notes. PTAL.
```
You may also refer to another example [here](https://github.com/skypilot-org/skypilot/tree/master/examples/spot/resnet_ddp) for periodically checkpointing with PyTorch.
```

Reply:
I think it's good to go!